components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1931,6 +1917,94 @@ void RewardsServiceImpl::SaveInlineMediaInfo(
                     std::move(callback)));
 }
 
+void RewardsServiceImpl::UpdateMediaDuration(
+    const uint64_t window_id,
+    const std::string& publisher_key,
+    uint64_t duration) {
+  if (!Connected()) {
+    return;
+  }
+
+  bat_ledger_->UpdateMediaDuration(window_id, publisher_key, duration);
+}
+
+void RewardsServiceImpl::GetPublisherInfo(
+    const std::string& publisher_key,
+    GetPublisherInfoCallback callback) {
+  if (!Connected()) {
+    return;
```

Comment:
this should callback as extension is expecting response

Reply:
you can easily crash browser if you do this:
1) clean profile (do not enable rewards)
2) load GL
3) restart browser
4) visit youtube.com
5) crash

why do we want to get publisher info if rewards is off?

Reply:
Fixed callback and added `rewards_enabled` precondition to `Greaselion.json` (it wasn't working when we started this project, but seems OK now).